### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

### CODE REVIEW

**Comment by billperegoy** [comment]
Thanks for the contribution. I'll get this merged and deployed later this week!

**Comment by djthread** [comment]
Awesome! Thank you very much for that, kianmeng. Curious - what made you go for the Apache license? I usually just go with MIT, but I'm not the most knowledgable on these things.

**Comment by kianmeng** [comment]
> Awesome! Thank you very much for that, kianmeng. Curious - what made you go for the Apache license? I usually just go with MIT, but I'm not the most knowledgable on these things.

@djthread It was Apache2 license all this while before my PR.

```
  defp package() do
    [
      licenses: ["Apache-2.0"],
      links: %{"GitHub" => "https://github.com/billperegoy/graphql_builder"}
    ]
  end
```

File: README.md [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+
+## Installation
+
+The package can be installed by adding `:graphql_builder` to your list of
+dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:graphql_builder, "~> 0.1.0"}
```
Comment (Ironjanowar) [inline README.md:27]: Maybe update the version to `0.3.0`?

Comment (kianmeng) [reply README.md]: @Ironjanowar Done.

**Comment by kianmeng** [comment]
@billperegoy Any updates on this?

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**